### PR TITLE
[report-converter] Support Pylint

### PR DESCRIPTION
**Pull request by csordasmarton**

### CODE REVIEW

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tools/report-converter/tests/unit/pylint_output_test_files/files/simple.py  1
- tools/report-converter/codechecker_report_converter/pylint/analyzer_result.py  2
- tools/report-converter/codechecker_report_converter/pylint/output_parser.py  7
- tools/report-converter/tests/unit/test_pylint_parser.py  3
         

Clones added
============
- tools/report-converter/tests/unit/test_pylint_parser.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=5215943&bid=17102492)

File: tools/report-converter/README.md [hunk 1]
```diff
@@ -246,6 +248,27 @@ report-converter -t eslint -o ./codechecker_eslint_reports ./eslint_reports.json
...
+
+The recommended way of running the Pylint tool is to generate a `json` output
+file.
+
+The following example shows you how to run Pylint and store the results found
+by Pylint to the CodeChecker database.
+
+```sh
+# Run Pylint.
+eslint -f json /path/to/my/project > ./pylint_reports.json
```
Comment (Xazax-hun) [inline tools/report-converter/README.md]: Is `eslint` intentional here?

Comment (csordasmarton) [reply tools/report-converter/README.md]: Thanks, good catch. I fixed it.

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tools/report-converter/codechecker_report_converter/pylint/analyzer_result.py  2
- tools/report-converter/tests/unit/test_pylint_parser.py  3
- tools/report-converter/codechecker_report_converter/pylint/output_parser.py  7
- tools/report-converter/tests/unit/pylint_output_test_files/files/simple.py  1
         

Clones added
============
- tools/report-converter/tests/unit/test_pylint_parser.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=5215943&bid=17102492)

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tools/report-converter/tests/unit/test_pylint_parser.py  3
- tools/report-converter/codechecker_report_converter/pylint/analyzer_result.py  2
- tools/report-converter/tests/unit/pylint_output_test_files/files/simple.py  1
- tools/report-converter/codechecker_report_converter/pylint/output_parser.py  7
         

Clones added
============
- tools/report-converter/tests/unit/test_pylint_parser.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=5215943&bid=17102492)

**Pull request merged**